ert3/console/_console.py

```diff
@@ -5,6 +5,8 @@
 import sys
 import yaml
 
+from ert3.exceptions import _exceptions as exceptions
```

Comment:
We should strive to not use `ert3` beyond the main namespace, so I suggest we remove this line, import the exception in `ert3/exceptions/__init__.py`, so that we further down can do `ert3.exceptions.Error`...